Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -185,7 +185,29 @@ class CallController {
             webRTCClient?.ownCapabilities = ownCapabilities
         }
     }
-    
+
+    /// Initiates a focus operation at a specific point on the camera's view.
```

Comment:
Having the same doc in 3 places seems a bit redundant, we only need it for the public method above.

Reply:
Ok, up to you on this one.